tests/types/array.test.js

```diff
@@ -59,6 +61,23 @@ describe("ArrayType", function() {
       });
     });
 
+    describe("sort", () => {
+      let sorted;
+
+      beforeEach(() => {
+        let unsorted = create(ArrayType, [...valueOf(ms)].reverse());
```

Comment:
There's no need to use the existing microstate. I think reversing it actually obfuscates the meaning of the test.

```js
let unsorted = create([String], ["c", "b", "a"]);
````
is much more clear.